### DISCUSSION TITLE
about fork

### DISCUSSION THREAD

**Discussion by yama1321** [community/community#118653] [Repositories] [answered]
Is there a fee for the ability to fork repositories on github?

**Comment by Akash1134** [comment, marked as answer]
Hi @Sangria1124 ,

Forking repositories on GitHub is free of charge. You can fork any public repository irrespective of whether you have a free or a paid GitHub account. However, to fork a private repository, you will need to have access permission to that repository.

For details on how to fork a repository, see [Fork a repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo)

Thanks!

**Reply by yama1321** [reply]
Thank you for your response. I really appreciate it!
